clients/src/test/java/org/apache/kafka/clients/consumer/RangeAssignorTest.java

```diff
@@ -178,6 +186,150 @@ public void testTwoConsumersTwoTopicsSixPartitions() {
         assertAssignment(partitions(tp(topic1, 2), tp(topic2, 2)), assignment.get(consumer2));
     }
 
+    @Test
+    public void testTwoStaticConsumersTwoTopicsSixPartitions() {
+        // although consumer 2 has a higher rank than 1, the comparison happens on
+        // instance id level.
+        String consumer1 = "consumer-b";
```

Comment:
Why do you not factor out also these consumer ids and the consumer ids below?

Reply:
Name them low and high to be more clear